packages/core/content-manager/shared/contracts/collection-types.ts

```diff
@@ -305,15 +305,21 @@ export declare namespace BulkPublish {
     body: {
       documentIds: Modules.Documents.ID[];
     };
-    query: {};
+    query: {
+      // If not provided, the default locale will be used
+      // Otherwise specify the locales to publish of the documents
+      locale?: string | string[] | null;
+    };
```

Comment:
Do we have some idea of how we are going to use this BulkPublish for multiple locales? 🤔 Because I was thinking in a API like:
```
body:

[ { documentId: 'abcd', locales: ['en'] }, { documentId: 'qwerty', locales: ['en', 'es'] } ]
```

So, in one action we can publish multiple entities in the locales we are interested for that specific entity

cc @Marc-Roig

Reply:
Do we have this use case in the CM? 

I was thinking that edit view bulk publish = single locale across multiple
documents whereas list view bulk publish (this PR) is multiple locales of the
same document.

Regardless I think passing multiple locales and documentIds should work in a way
like:

"publish all of these locales of all of these documents if they exist"

e.g. for documents that don't have some of the locales passed they'd just be
ignored

What do you think?

Reply:
I agree, I wasn't considering the new bulkPublishs options we have in the editView